modules/standard/FileSystem.chpl

```diff
@@ -50,7 +52,7 @@ proc chdir(out err: syserr, name: string) {
    Note: this is not safe within a parallel context.  A chdir call in one task
    will affect the current working directory of all tasks for that locale.
 */
-proc chdir(name: string) {
+proc locale.chdir(name: string) {
   var err: syserr = ENOERR;
   chdir(err, name);
```

Comment:
You don't need to specify a "this" on another one of your own methods.  And since there isn't another chdir, it doesn't get confused.  (I tested to be certain it was looking at the right one).